code/game/gamemodes/machinegod/machinegod_items.dm

```diff
@@ -0,0 +1,427 @@
+/obj/item/clock_component
```

Comment:
As a general comment, combining all this stuff together in one file makes it pretty hard to read.
The spells should go in the spell folder, and then I think you should split this into components and equipment.